src/content/docs/infrastructure/host-integrations/installation/install-infrastructure-host-integrations.mdx

```diff
@@ -40,83 +39,74 @@ import osUbuntu from 'images/os_icon_ubuntu.webp'
 
 import osDebian from 'images/os_icon_debian.webp'
 
-New Relic's [on-host integrations](/docs/integrations/host-integrations/getting-started/introduction-host-integrations) are a type of infrastructure integration that collect data from core services running on your hosts, such as MySQL, Apache, or Redis, among others.
+These are general install instructions for our infrastructure agent on-host integrations (those that work with our infrastructure agent), and technical details about how they work. 
```

Comment:
@zuluecho9 You might be able to streamline this introduction by ditching that h3 (Integrations that work in tandem with our infrastructure agent) and just inserting that wording right in the intro:

Some of our integrations work in tandem with our infrastructure agent, which means you'll install an integration package along with our infrastructure agent. 

Here is a list of  with our infrastructure agent integrations we support: (I'd suggest we not have any links here because we don't want users to click away from this page. You might just have a table of apps without links. Then, later in the doc after the installation, have a configuration section that includes those links to the specific pages which have configuration steps.